manual/development-process/access-integration-db.md

```diff
@@ -0,0 +1,78 @@
+#Accessing the Integration database
```

Comment:
I think these instructions should work for any environment. It's just that production access should only be used in an emergency, if we can't debug something using our usual logs.